components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -179,51 +179,86 @@ func runShowCmd(cmd *cobra.Command, args []string) error {
-		sshPort := infra.Outputs.SSHPort.Value
+		var outFrontend string
+		var outBackend string
+		var outFiles []string
+		if len(args) > 0 && args[0] != "" {
+			outFrontend = args[0]
+			outBackend = "> " + args[0]
+			outFiles = []string{outFrontend}
+		}
+		frontendCommand := fmt.Sprintf(GET_FRONTEND_CONFIG, outFrontend)
+		postgresqlCommand := fmt.Sprintf(GET_BACKEND_CONFIG, "postgresql", outBackend)
+		opensearchCommand := fmt.Sprintf(GET_BACKEND_CONFIG, "opensearch", outBackend)
+
+		frontend := &Cmd{
+			CmdInputs: &CmdInputs{
+				NodeType: false,
+			},
+		}
 
-		sshConfig := &SSHConfig{
-			sshUser:    sshUser,
-			sshKeyFile: sskKeyFile,
-			sshPort:    sshPort,
+		automate := &Cmd{
+			CmdInputs: &CmdInputs{
+				Cmd:         frontendCommand,
+				Single:      false,
+				InputFiles:  []string{},
+				Outputfiles: outFiles,
+				NodeType:    configCmdFlags.automate,
+			},
 		}
-		sshUtil := NewSSHUtil(sshConfig)
 
-		if configCmdFlags.overwriteFile {
-			writer.Errorln("Overwrite flag is not supported in HA")
+		chef_server := &Cmd{
+			CmdInputs: &CmdInputs{
+				Cmd:         frontendCommand,
+				Single:      false,
+				InputFiles:  []string{},
+				Outputfiles: outFiles,
+				NodeType:    configCmdFlags.chef_server,
+			},
 		}
 
-		var scriptCommand string
-		var hostIpArray []string
-		switch true {
-		case configCmdFlags.automate:
-			hostIpArray = infra.Outputs.AutomatePrivateIps.Value
-			scriptCommand = GET_FRONTEND_CONFIG
-		case configCmdFlags.chef_server:
-			hostIpArray = infra.Outputs.ChefServerPrivateIps.Value
-			scriptCommand = GET_FRONTEND_CONFIG
-		case configCmdFlags.postgresql:
-			hostIpArray = infra.Outputs.PostgresqlPrivateIps.Value
-			scriptCommand = fmt.Sprintf(GET_CONFIG, "postgresql")
-		case configCmdFlags.opensearch:
-			hostIpArray = infra.Outputs.OpensearchPrivateIps.Value
-			scriptCommand = fmt.Sprintf(GET_CONFIG, "opensearch")
-		default:
-			return errors.New("Missing or Unsupported flag\n Please run the following command to see all available flags \n\n`chef-automate config show --help`\n")
+		postgresql := &Cmd{
+			CmdInputs: &CmdInputs{
+				Cmd:         postgresqlCommand,
+				Single:      false,
+				InputFiles:  []string{},
+				Outputfiles: outFiles,
+				NodeType:    configCmdFlags.postgresql,
+			},
 		}
 
-		for i := 0; i < len(hostIpArray); i++ {
-			sshUtil.getSSHConfig().hostIP = hostIpArray[i]
-			output, err := sshUtil.connectAndExecuteCommandOnRemote(scriptCommand, true)
-			if err != nil {
-				return err
-			}
-			writer.Success("Configuration from " + hostIpArray[i] + " node:\n")
-			writer.Println(output)
+		opensearch := &Cmd{
+			CmdInputs: &CmdInputs{
+				Cmd:         opensearchCommand,
+				Single:      false,
+				InputFiles:  []string{},
+				Outputfiles: outFiles,
+				NodeType:    configCmdFlags.opensearch,
+			},
 		}
 
-		return nil
+		nodeMap := &NodeTypeAndCmd{
+			Frontend:    frontend,
+			Automate:    automate,
+			Chef_server: chef_server,
```

Comment:
Rename this `chef_server` to camel case.

Reply:
Done